src/reporting/lambda/custom-resource/quicksight/index.ts

```diff
@@ -295,11 +302,37 @@ const createQuickSightDashboard = async (quickSight: QuickSight,
     },
   };
 
+  const folder = await quickSight.createFolder({
+    AwsAccountId: commonParams.awsAccountId,
+    FolderId: `clickstream_${commonParams.databaseName}_${commonParams.schema}`,
```

Comment:
Let's move it to a utility for cross-project sharing.

Reply:
We have similar usage in the backend's quicksight implementation. Do we have to share the code with the backend API? @tyyzqmf

Reply:
Use [QuickSight](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-quicksight/Class/QuickSight/) . This way of calling services is more elegant by directly using 'QuickSightClient'.